internal/resolution/variablesources/bundle_uniqueness_test.go

```diff
@@ -17,6 +20,255 @@ import (
 	"github.com/operator-framework/operator-controller/internal/resolution/variablesources"
 )
 
+func TestMakeBundleUniquenessVariables(t *testing.T) {
```

Comment:
Is this the most minimal test we can create? There's a *lot* of setup here and it's not entirely clear to me that all of it is necessary - for instance, if you need to test dependencies through GVK and package name, it makes sense to create minimal setups for each. It's hard to follow the independent threads here and the size of the test dataset makes it difficult to ascertain relationships between entities.

Reply:
Hm. I  took the test data from the old test (it is still below, but I copied it over so it is easier to clean it up later in #501). But it looks like we have some test data related to GVK uniqueness in here which we probably don't need at the moment.

Tomorrow I'll look into whether we can reduce the test data.

Background: we had some dead code related to GVK uniqueness and some tests for it. Code got removed, but it looks like data was not cleaned up.

Reply:
@stevekuznetsov I think this was a really good catch! I had a closer look at the test data and there was quite a lot of junk related to GVK uniqueness which we do not enforce at the moment. Now we are down to 3 bundles.

I also added some comments to the test to explain why we need test data to meet certain requirements. Hopefully it will help future readers.